website/pages/docs/commands/operator/snapshot-agent.mdx

```diff
@@ -0,0 +1,149 @@
+---
+layout: docs
+page_title: 'Commands: operator snapshot agent'
+sidebar_title: snapshot agent
+description: |
+  Periodically saves snapshots of Nomad server state
+---
+
+# Command: operator snapshot agent
+
+<EnterpriseAlert />
```

Comment:
This doesn't render for me locally but I noticed that consul uses it in for the banner in https://www.consul.io/docs/commands/snapshot/agent - will ask docs folks.

Reply:
@angrycub had me do this:
```
~> **Enterprise Only!** This functionality only exists in Nomad
Enterprise. This is not present in the open source version of Nomad.
```

in https://github.com/hashicorp/nomad/blob/master/website/pages/docs/job-specification/multiregion.mdx

Reply:
I think I see why: we don't have https://github.com/hashicorp/nomad/pull/7910 merged, which has that fancy new component to use. 😁

Reply:
I *think* the reason Jeff didn't merge that was because he thought through a different abstraction which I implemented in my commit on this PR.